### DISCUSSION TITLE
Can I get chia reward if the wallet is not synced but the node is synced and it is farming

### DISCUSSION THREAD

**Discussion by chenus1** [Chia-Network/chia-blockchain#7743] [Support]
Can I get chia reward if the wallet is not synced but the node is synced and it is farming?

My  system is window 8.1 pro with latest chia 1.1.6 which was working. However I added another wallet key then it is messed up. Not able to get wallet synced more. I tried to update chia to 1.1.7 to 1.2.1, which can get wallet syncing, but not able to get harvester machines connected (the connection to harvester machines flashing on and off and not plots can pass the filter appears on the Window GUI.

**Comment by rmica** [comment]
Yes, you get rewards even with a turned off wallet.

**Reply by chenus1** [reply]
thanks!
